echopype/tests/calibrate/test_cal_params_integration.py

```diff
@@ -79,6 +79,8 @@ def test_cal_params_intake_EK60(ek60_path):
     cal_obj = ep.calibrate.calibrate_ek.CalibrateEK60(echodata=ed, env_params=None, cal_params=None, ecs_file=None)
 
     # Check cal params ingested from both ways
+    # Need to drop ping_time for cal_obj.cal_params since get_vend_cal_params_power are related to
```

Comment:
Not so much being picky as confirming that something isn't missing in this comment.
```suggestion
    # Need to drop ping_time for cal_obj.cal_params since get_vend_cal_params_power is related to
```